### ISSUE TITLE
Le stockage local des données ne gère pas les pertes en cas de fork

### ISSUE THREAD

**Issue by c-geek** [duniter/cesium#113]
Cela vient tout juste de m'arriver : Remuniter m'envoie une récompense pour des blocs calculés, récompense que je vois d'abord en attente dans Cesium, puis validée. Jusque là tout va bien.

Puis étonnamment, quelques heures plus tard, Remuniter me renvoie *exactement* la même transaction, au blockstamp près. Cesium continue de m'indiquée la transaction précédemment reçue, mais *en plus* la quasi-même qui se trouve répétée à cause d'un fork (Remuniter détecte que sa transaction n'est pas présente en blockchain).

En réalité, si je me connecte à Cesium depuis une fenêtre de navigation privée, alors ma 1ère récompense a disparue, ce qui est correct du fait du fork.

**Comment by blavenie** [comment]
Oui, je vais commencer à stocker la version de la blockchain, pour détecter les rollback
